jpype/_jcollection.py

```diff
@@ -134,6 +137,45 @@ def count(self, obj):
         except TypeError:
             return 0
 
+    def insert(self, idx, obj):
+        if idx<0:
+            idx += self.size()
+        return self.add(idx, obj)
+
+    def append(self, obj):
+        return self.add(obj)
+
+    def reverse(self):
+        _jpype.JClass("java.util.Collections").reverse(self)
```

Comment:
This impl is an inplace operation, right? I thought reverse() just returns an reversed iterator, but leaves the original sequence untouched.

Reply:
It is inplace. Sorry for the noise